cluster/README.md

```diff
@@ -129,3 +129,57 @@ $ ls -l gitops_repo_key*
 
 Flux currently requires write access to the git repository to store reconcilation state. For Github, the process to add a deploy key is documented 
 [here](https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/).
+
+### Terraform State
+
+When one isses a `terraform apply`, Terraform records the information about what is created in a [Terraform state file](https://www.terraform.io/docs/state/).  By default, the directory in which `terraform apply` is executed in will create a file named `terraform.tfstte`.  Terraform needs this information to know the state of the cluster for future modifications to the infrastructure.
+
+In devops / production scenarios, storing the state file on a local file system may not be the desired scenario.  Terraform supports storing state remotely.  One of those possible locations is in an Azure Blob Store.  This is defined using a `backend` block.  The basic block looks like:
```

Comment:
It is never recommended because you will have conflicts and problem if two people try to make any changes at the same time.